Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -170,6 +170,10 @@ class OWTestLearners(OWWidget):
     TARGET_AVERAGE = "(Average over classes)"
     class_selection = settings.ContextSetting(TARGET_AVERAGE)
 
+    shown_scores = \
+        settings.Setting({"AUC", "CA", "F1", "Precision", "Recall",
```

Comment:
yes, probably not, no

If this change would "break" json serialization, I would say yes, but since it is already broken, I would say leave it like it is.